### ISSUE TITLE
Motion & Fitness Icon And Other Settings Icons

### ISSUE THREAD

**Issue by throughtheiris25** [AnemoneTeam/Anemone#213]
Trying to theme the "Motion & Fitness" Icon under Privacy? Any ideas what bundle and png names need to be. Have all the settings icons themed perfectly, including Motion, but this icon, I have no idea. 

Also 'Home' icon under iCloud does not theme, although HomeData icon in accounts is present. None of the icons under "Itunes & App Store" theme.

Have tried all suggestions I could find elsewhere, on your WIKI, and reddit.

Any advice would be great.

**Comment by IrishMan19** [comment]
Motion@2/3x.png

**Comment by throughtheiris25** [comment]
Motion@2/3x.png fully themed under com.apple.preferences-ui-framework.

**Comment by throughtheiris25** [comment]
Found 'Games' and 'Wallet' under Notifications section in Settings not being themed either.

**Comment by throughtheiris25** [comment]
I think that these icons cannot be themed. Six themes, including the one they say has the best icon compatibility, and none of these icons theme. 

To list them:

Settings>Privacy> (Motion & Fitness) icon
Settings>iCloud> (Home) icon
Settings>iTunes & App Store > (All 4) icons
Settings>Notifications> (Games) and (Wallet) icons

That's what I can find yet, and have no idea if its just an obscure Bundle ID/Folder that no one knows about, or whether the themeing engine does not accommodate them. 

Any help would be great, and please remember, I have tried all the methods available. No one can confirm that these icons theme anywhere I've asked, so maybe a bug.

**Comment by IrishMan19** [comment]
For Settings/iCloud you need to create com.apple.accountsui folder with com.apple.Dataclass.Backup@2/3x.png, com.apple.Dataclass.Bookmarks@2/3x.png, com.apple.Dataclass.Calendars@2/3x.png......

For Motion&Fitness you need Motion@2/3x.png in  com.apple.preferences-framework and com.apple.preferences-ui-framework folders

**Comment by throughtheiris25** [comment]
iCloud section is fine, and the icon replacement (Motion@2/3x.png) in both apple.preferences folders has already been done. I mentioned this in a previous post. I've done everything, checked everywhere. I have +7000 icons files in one of the themes, and these icons are not themed. Check the icons I mentioned on your phone, as I described, and check if they are themed on your phone. They probably aren't.

**Comment by IrishMan19** [comment]
the only icons i dont have themed are those 4 icons under Settings/iTunes&AppStore.

**Comment by throughtheiris25** [comment]
Weird, I don't understand why the others are all showing for you. I have them all themed perfectly, Glaskart covers everything, yet those other ones I mentioned don't work. Did you check them all out?

Anyhow, are u on IOS10?

**Comment by IrishMan19** [comment]
i6S 10.2

**Comment by throughtheiris25** [comment]
Makes no sense. Name the theme your using, so I can check that all files are comparable on my end, wanna buy another theme, so maybe your choice is good.

**Comment by IrishMan19** [comment]
The theme im using is not released yet but you can check my other theme, Red Gold on Cydia. ( dont pirate, haha)

**Comment by throughtheiris25** [comment]
I'm just a few other big ones, with a lot of icons first, although that's a nice theme. The color scheme isn't really my thing. I'm gonna check some places to check your themes file list just to compare anyway. Maybe none of these themes have a specific bundle or something. Thanks for at least checking for me! At least I know it's not a bug!

**Comment by IrishMan19** [comment]
Bundles: 
1. com.apple.GameCenterUI
Icons:  GKEventIcon@3x.png (180x180)
            GKEventIcon@2x.png (120x120)

2. com.apple.ios.StoreKitUI

Icons: ProductViewGameCenterBadge@3x.png (72x72)
           ProductViewGameCenterBadge@2x.png (58x58)

More icons in those folders but those are for GameCenter

**Comment by throughtheiris25** [comment]
Thanks for that. At least a place to start. No one has given me any useful advice thus far. That'll at least rid me of one of them. Got the Home one sorted myself. Under com.apple.accountsui, changed com.apple.Dataclass.HomeData@2x/3x to com.apple.Dataclass.Home@2x/3x and that worked. Was just a guess. May help you if your having that problem under iCloud.

Now that dam Wallet icon. There a Wallet bundle u can think of off your head? Have all the other wallet items done, under preferences-ui and iconbundles.

**Comment by IrishMan19** [comment]
ProductViewPassbookBadge@2x.png and ProductViewPassbookBadge@3x.png goes to com.apple.ios.StoreKitUI (same size as Game Center icons)

**Comment by throughtheiris25** [comment]
Coolio man, I'll give those a wirl when I get home. Thanks a ton. That StoreKitUI part is probably my problem. iTunes & App Store we can just assume is broken!

**Comment by IrishMan19** [comment]
i forgot 2 more for GameCenter.... AppleIDGameCenter@3x.png(120x120) and AppleIDGameCenter@2x.png(80x80) in com.apple.purplebuddy bundles

**Comment by throughtheiris25** [comment]
LOL! Game Center. Does anyone even use GameCenter. Lord! Any other Motion stuff you can think of, because as I said those other two items under Preference-UI don't work.

**Comment by IrishMan19** [comment]
Those 4 in com.apple.preferences-framework and com.apple.preferences-ui-framework is all i have for Motion.

**Comment by throughtheiris25** [comment]
Ok, I'll get to messing around later! Should fix at least a few of them. You are a legend bro, thanks for the help!

**Comment by IrishMan19** [comment]
Glad to help, bud

**Comment by throughtheiris25** [comment]
So, managed to fix Motion and Home issues. Those dam Wallet and Games icons in Notifications section are still messed up. Themed every suggested Icon, including those you mentioned, plus anything else I could find in Filza. They remain beautifully a pain. Haha!

**Comment by dreamnet** [comment]
`Settings -> Notifications -> Wallet` is
`com.apple.preferences-ui-framework/Wallet@2x.png` and
`com.apple.preferences-ui-framework/Wallet@3x.png` on iOS9.

`Settings -> Notifications -> Game Center` must be themed with the IconBundles-method,
`IconBundles/com.apple.gamecenter-large.png`.

**Comment by throughtheiris25** [comment]
All six of my themes have those icons themed and in same mentioned specs as system folders, but those two icons remain unthemed with all these themes. Found another theme which does theme them, so trying to figure out what the problem may be. Maybe something stupid though, or a change in iOS 10. No idea yet.

Then the next thing, theming Share Sheets. Whoopy. Any idea how to theme 3rd party specific Share Sheets, like Whatsapp Forwarding messages for instance. Where do you look for such things, when no Bundle IDs exist for an action. Wiki is very specific to very specific things.

**Comment by matruka** [comment]
Have you guys all Icons themed now in Settings?
I have ALL themed except one Icon under the iCloud Settings... the News Icons. Any Idea how to theme it?

**Comment by throughtheiris25** [comment]
I managed to fix some of my icons by restarting my device strangely enough, so the advice given above is probably correct, and you may need a full restart and re-jailbreak. Check 'com.apple.Dataclass.News.png', 'com.apple.Dataclass.News@2x.png' and 'com.apple.Dataclass.News@3x.png' under com.apple.accountsui bundle. Then re-apply in anemone, respring, and restart if necessary.

**Comment by matruka** [comment]
you have those themed? can you show a picture?
i have exactly these images and it's not theming for me... tried everything, reapply, respring, recache, restart ... 

here the discussion https://twitter.com/Matrukaios/status/838473711867080704 several people couldn't theme this icon

**Comment by throughtheiris25** [comment]
If that doesn't work, then no idea man. Those icons are themed in all the themes I downloaded, the Home icon in iCloud was the one I had a problem with. Had to rename the file. I just checked which icon it appeared to be on my end that changes that icon specifically, and that was the only one in the associated folder that meets the description. No picture required man, because the 3 icons I mentioned seem to be what themes the news icons in that section. Maybe it doesn't, I'm just guessing with my limited experience. Maybe one of the others guys will respond with a better answer.
